install/configuring_inventory_file.adoc

```diff
@@ -1104,6 +1104,25 @@ openshift_hosted_routers=[{'name': 'router1', 'certificate': {'certfile': '/path
 
 include::install/topics/glusterfs_intro.adoc[]
 
+You configure {gluster} clusters using variables, which interact with the {product-title} clusters.
+The variables reside in the `[OSEv3:vars]` group and include host variables, role
+variables, and image name and version tag variables.
+
+The `glusterfs_devices` variable is a host variable that defines the list of block devices
+that can be managed as part of the {gluster} cluster. Each host in your configuration must
+have at least one `glusterfs_devices` variable, and for every configuration, there must be at
+least one bare device with no partitions or LVM PVs.
+
+Role variables control the integration of a {gluster} cluster into a new or existing {product-title}
+cluster. You can define a number of role variables, each of which also has a
+corresponding variable to optionally configure a separate {gluster} cluster
+for use as storage for an integrated Docker registry.
+
+Image name and version tag variables are used to prevent {product-title} pods from
```

Comment:
Is this the only purpose of these variables? I'd say something like "Define image name and version tag variables to prevent..."

Reply:
Rewritten. Thanks.